examples/Demo/Shared/Pages/Listbox/Examples/ListboxWithOptionTemplate.razor

```diff
@@ -0,0 +1,17 @@
+﻿@inject DataSource Data
+
+<FluentListbox Items="@Data.People"
+               OptionValue="@(i => i.PersonId.ToString())">
+    <OptionTemplate>
+        <FluentIcon Icon="Icons.Regular.Size16.Delete" Slot="end" OnClick="@(() => DemoLogger.WriteLine($"{@context.LastName} deleted"))" />
```

Comment:
Why use the "deleted" icon? Without the code to remove this item from the list of people, I'm afraid we're going to get questions about it. Perhaps a more "neutral" example with a "People" illustration would be more appropriate?